### DISCUSSION TITLE
Porting to CTMON65

### DISCUSSION THREAD

**Discussion by justlostintime** [cc65/cc65#1981] [General]
I followed the guidlines and created the inc,h and cfg files, created the ctmon65 directory and updated the makefile for target the ctmon65, it created all the directories but gives me this error?
Make[1]: Nothing to be done for 'gamate'.
make[1]: Nothing to be done for 'kim1'.
ctmon65 - common/_afailed.c
cc65: Unknown target system 'ctmon65'
make[1]: *** [Makefile:285: ../libwrk/ctmon65/_afailed.o] Error 1
make: *** [Makefile:121: ctmon65] Error 2

I must be missing something, but just can not figure it out!

Can anyone help me?

**Comment by mrdudz** [comment]
not sure what the question is, but if you want to add a target that can be used like the regular targets (with -t <target>) then you'll have to add this target to the assembler/compiler as well

**Comment by polluks** [comment]
@justlostintime
please follow https://cc65.github.io/doc/customizing.html

**Reply by justlostintime** [reply]
Thanks will read through it again

**Comment by justlostintime** [comment]
I got it working, adding all the io library entries.
Is it possible to submit a merge request with this library
